MIGRATION-FROM-SCK.md

```diff
@@ -0,0 +1,345 @@
+
+All SCK applications use Fluentd to work with logs, metrics and objects. Fluentd has significant performance issues when used to fetch logs in a Kubernetes cluster with pods that have very high throughput.
+
+Splunk OpenTelemetry Collector for Kubernetes provides significant performance improvements over the SCK through use of the OpenTelemetry Collector agent and native OpenTelemetry functionality for logs collection rather than Fluentd. You can learn more about the performance characteristics of this new application [Performance of native OpenTelemetry logs collection](https://github.com/signalfx/splunk-otel-collector-chart/blob/main/docs/advanced-configuration.md#performance-of-native-opentelemetry-logs-collection).
+
+Splunk OpenTelemetry Collector for Kubernetes has the following components and applications:
+
+1. Splunk OpenTelemetry Collector Agent (`agent`) to fetch logs, metrics and traces from a kubernetes cluster (deployed as a Kubernetes DaemonSet)
+2. Splunk OpenTelemetry Collector Cluster Receiver (`clusterReceiver`) to fetch metrics from a Kubernetes API (deployed as a Kubernetes 1-replica Deployment).
+3. Optional Splunk OpenTelemetry Collector Gateway (`gateway`) to forward data through it to reduce load on Kubernetes API and apply additional processing (deployed as a Kubernetes Deployment).
+
+There is no application available for fetching Kubernetes objects metadata from a Kubernetes cluster.
+
+### Changes in logs, metrics, and objects
+
+| Logging | Metrics | Objects |
+|---|---|---|
+| Redhat UBI docker images for our applications are no longer available, as we now use scratch images | The naming convention of the metrics used in Splunk OpenTelemetry Collector for Kubernetes has changed and there are minor differences in the names of the metrics | Not implemented or available |
+| AWS FireLens is not supported. | Changes in the number of metrics (fewer metrics available) and additional metrics for the OpenTelemetry Collector |  |
+|  |  |  |
+
+### Changes in logs in Splunk OpenTelemetry Connector for Kubernetes
+
+* Red Hat Universal Base Image (UBI) Docker images for our applications are no longer available, as we now use scratch images.
+* AWS FireLens is not supported.
+
+### Changes in metrics in Splunk OpenTelemetry Connector for Kubernetes
+
+* The naming convention of the metrics used in Splunk OpenTelemetry Collector for Kubernetes follows the OpenTelemetry specification and is different than SCK. You will observe minor differences in the names of the metrics.
+* Previously in SCK, you could get a large number of metrics from various APIs for kubernetes. However, in recent versions of kubernetes 1.18+, these API sources are disabled by default and there are fewer metrics available. The previous list of metrics can be found [Metrics Information](https://github.com/splunk/fluent-plugin-kubernetes-metrics/blob/master/metrics-information.md).
```

Comment:
See [Metrics Information](https://github.com/splunk/fluent-plugin-kubernetes-metrics/blob/master/metrics-information.md) for the previous list of metrics.